### ISSUE TITLE
SQLite view

### ISSUE THREAD

**Issue by pelord** [zer0infinity/OGR2GUI#2]
OGR2GUI don't list SQLite view.

**Comment by zer0infinity** [comment]
Hi pelord,

Would you mind to upload a SQLite file example?

Thanks in advance.

**Comment by pelord** [comment]
Yes, my dataset is too large, i will subset it en upload it tomorrow
